power_grid_model_c/power_grid_model/include/power_grid_model/main_model_impl.hpp

```diff
@@ -57,6 +57,64 @@ template <> struct output_type_getter<SolverOutput<asymmetric_t>> {
     template <class T> using type = meta_data::asym_output_getter_s<T>;
 };
 
+struct power_flow_t {};
+struct state_estimation_t {};
+struct short_circuit_t {};
```

Comment:
What are these supposed to mean exactly?

Reply:
these are tags used by the (runtime) function selector to choose the individual implementation functions for the different calculation types

Reply:
So `t` stands for tags. Can we call them `_tag` directly? `_t` implies something different to me.

Reply:
Cfr `symmetric_t` vs `symmetry_tag`: `_t` stands for the concrete type while `_tag` stands for the "it is something that looks like a xyz...", i.e. the concept that acts upon a tag type.

We can talk about the conventions but i think that's a whole separate topic altogether. I'd propose to keep it according to the existing conventions in this PR